macros/dmt_unit_test.sql

```diff
@@ -1,7 +1,7 @@
-{% test unit_test(model, input_mapping, expected_output, name, description) %}
-    {% set test_sql = dbt_datamocktool.get_unit_test_sql(model, input_mapping) %}
+{% test unit_test(model, input_mapping, expected_output, depends_on, name, description,compare_columns) %}
```

Comment:
Could you remove the references to `depends_on` here since it's a separate PR? (I'm reviewing that one next, but let's keep them separate)